src/modules/showCount.js

```diff
@@ -0,0 +1,3 @@
+const showsNumber = (list) => list.length;
```

Comment:
- [ ] Kindly make sure to add a function to show the items on the home page.

![image](https://user-images.githubusercontent.com/93630700/205456891-71af43af-0104-4f95-8550-5374ffd1de77.png)